contracts/contracts/strategies/balancer/BaseAuraStrategy.sol

```diff
@@ -80,7 +80,9 @@ abstract contract BaseAuraStrategy is BaseBalancerStrategy {
         onlyHarvester
         nonReentrant
     {
-        // Collect BAL and AURA
+        /* Similar to Convex calling this function collects both of the
```

Comment:
Needs a comma after "Similar to Convex".

Reply:
thanks, added